traefik/metadata.csv

```diff
@@ -0,0 +1,2 @@
+traefik.total_status_code_count,gauge,,,,total count for each returned status code,,traefik,traefik
```

Comment:
Hi @ofek,

Thanks for reviewing this PR.
I left some fields empty because I did not know what value to set. I can set the name of the field if a value is needed.